src/worker/core/player/genRatings.hockey.ts

```diff
@@ -198,6 +198,37 @@ const genRatings = (
 		pots: { ...defaultOvrsOrPots },
 	};
 
+	// Youngest prospects !== 18 will be scaled, scaling stops at age 13 and 26
+	let age = g.get("draftAges")[0];
+	age = helpers.bound(age, 13, 26);
```

Comment:
Why different than other sports?

Reply:
When I left it the same as BBGM/FBGM, there were an absurd absurd number of superstars. Players below the min scaling were also a bit too good.